docs/lcb/notes/feature-objc_delegates.md

```diff
@@ -0,0 +1,115 @@
+# LiveCode Builder Language
+## Objective-C delegate support
+Functions have been added which allow the creation of custom delegate objects with LCB implementations of protocol methods.
+
+These are implemented in libfoundation, so can be bound to using the 
```

Comment:
Can we add LCB wrappers to the objc module?

Reply:
ah, good idea :-)